.gitignore

```diff
@@ -97,3 +97,8 @@ ENV/
 *.iml
 out
 gen
+
+# Test files
+save_test1.txt
+save_test2.txt
+save_test3.txt
```

Comment:
Instead of this, remove them at the end of the test file like other packages.

Reply:
Fixed in https://github.com/sepandhaghighi/mycoffee/pull/101/commits/a1374455d6e857409fbd7f6c6406948718af8ade